GitCommands/Settings/FileSettingsCache.cs

```diff
@@ -151,21 +152,28 @@ protected override void SaveImpl()
                     {
                         File.Copy(SettingsFilePath, backupName, true);
                     }
-                    catch (IOException)
+                    catch (Exception)
                     {
                         // Ignore errors for the backup file
                     }
                 }
 
-                // ensure the directory structure exists
-                var parentFolder = Path.GetDirectoryName(SettingsFilePath);
-                if (!Directory.Exists(parentFolder))
+                try
                 {
-                    Directory.CreateDirectory(parentFolder);
-                }
+                    // ensure the directory structure exists
+                    var parentFolder = Path.GetDirectoryName(SettingsFilePath);
+                    if (!Directory.Exists(parentFolder))
+                    {
+                        Directory.CreateDirectory(parentFolder);
+                    }
 
-                File.Copy(tmpFile, SettingsFilePath, true);
-                File.Delete(tmpFile);
+                    File.Copy(tmpFile, SettingsFilePath, true);
+                    File.Delete(tmpFile);
+                }
+                catch (Exception ex)
+                {
+                    MessageBox.Show(ex.Message, "Cannot save settings", MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
```suggestion
                    MessageBox.Show(ex.Message, "Cannot save settings", MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Reply:
changed - I felt informing was sufficient

Reply:
When showing errors it needs to be error, or if it isn't terminal, it could be a warning.